maintainers/maintainer-list.nix

```diff
@@ -7598,6 +7598,12 @@
     githubId = 1699155;
     name = "Steve Elliott";
   };
+  stephank = {
+    email = "nix@stephank.nl";
+    github = "stephank";
+    githubId = 89950;
+    name = "Stéphan Kochen";
+  };
```

Comment:
I've seen reviewers preferring this to be a separate commit.
(the commit tag line has a format when adding new maintainers)